### PR TITLE
updated the API  contract for the route GET tasks/self

### PR DESCRIPTION
**Pull request by Rucha1499**
- Updated the description of the route `tasks/self` in the contract.
- Added the Query `Completed=[boolean]`.

### CODE REVIEW

File: tasks/README.md [hunk 1]
```diff
@@ -65,10 +65,12 @@
 
 ## **GET /tasks/self**
 
-Returns all the tasks of the user.
+Returns all the completed tasks of user if query `Completed=true` is passed, else returns all the active and blocked tasks of the user.
 
 - **Params**  
   None
+- **Query**  
+  Completed=[boolean]
```
Comment (sumitd94) [inline tasks/README.md:73]: should be lowercase `completed=true`

**Review by sumitd94 — CHANGES_REQUESTED** [review]
One small fix and then looks good.

File: tasks/README.md (hunk 1 above)
Comment (Rucha1499) [reply tasks/README.md]: Done with the changes. Thank you! 😄

**Review by sumitd94 — APPROVED** [review]
Thanks for the changes !

**Pull request merged**